test/readrows.ts

```diff
@@ -71,8 +71,14 @@ describe('Bigtable/ReadRows', () => {
     service = new BigtableClientMockService(server);
   });
 
+  function setTestTimeout(test: mocha.Context) {
```

Comment:
```suggestion
  // helper function because some tests run slower
  // on Windows and need a longer timeout
  function setWindowsTestTimeout(test: mocha.Context) {
```